env_support/cmake/custom.cmake

```diff
@@ -1,3 +1,6 @@
+set(LVGL_VERSION "9.0.0")
```

Comment:
We have the LVGL version already in https://github.com/lvgl/lvgl/blob/master/lvgl.h#L16
so if it is possible, we should have the version number only in 1 place.

Other question:
is this change working in Windows environment, too?

Reply:
> so if it is possible, we should have the version number only in 1 place.

Should be done in version.h , for both lvgl.h and the packageconfig lvgl.pc.in .

> is this change working in Windows environment, too?

I don't know, I have no such platform available.

Reply:
Seems like for Makefile builds, this needs a static version.h, CMake can generate one.